lib/README.md

```diff
@@ -4,11 +4,11 @@ For python developers, client libraries are supplied for testing and development
 
 Name                                | Description
 ------------------------------------| -------------
-vapi_runtime	                    | vAPI runtime responsible for serialization/de-serialization of objects and wire protocol
-vcenter_bindings                    | Client stubs for vSphere Automation APIs
+vmware_vapi_runtime	                | vAPI runtime responsible for serialization/de-serialization of objects and wire protocol
```

Comment:
We do not have the library or source here. No need to add them in the library readme.
Applies to vmware_vcenter and the common client as well

Reply:
Done, removed